DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -50,6 +51,39 @@
 MAX_STORAGE_SIZE = 0
 USE_TOKENS = False
 
+def getDiskSpace(path, size = 'TB', total = False):
+    """
+      Returns disk usage of the given path.
+      If no size is specified, terabytes will be used by default.
+      If total is set to true, the total disk space will be returned instead.
+    """
+
+    if size.upper() == "KB":
```

Comment:
and it is also much better to not define the units in floating point but as integers!

MB: 1024*1024
GB: 1024*1024*1024
etc...